apps/api/src/app/widgets/usecases/update-subscriber-preference/update-subscriber-preference.usecase.ts

```diff
@@ -75,6 +78,13 @@ export class UpdateSubscriberPreference {
       }
     );
 
+    this.analyticsService.track('Update User Preference - [Notification Center]', command.organizationId, {
+      _organization: command.organizationId,
+      _subscriber: command.subscriberId,
+      _template: command.templateId,
+      ...updatePayload,
```

Comment:
No need to have the payload here

Reply:
Wouldn't we want to know what was enabled/disabled?